scripts/deploy/pre-deployment.sh

```diff
@@ -106,7 +106,7 @@ sed \
 	-e "s|^sleeper.subnet=.*|sleeper.subnet=${SUBNET}|" \
 	-e "s|^sleeper.tags.file=.*|sleeper.tags.file=${TAGS}|" \
 	-e "s|^sleeper.table.properties=.*|sleeper.table.properties=${TABLE_PROPERTIES}|" \
-	-i ${INSTANCE_PROPERTIES}
+	-i'.bu' ${INSTANCE_PROPERTIES}
```

Comment:
In scripts/dev/updateVersionNumber.sh we use `-i ""` to avoid this failing on Mac. Can we be consistent with this approach?

Reply:
Will update in next commit